.github/workflows/ci.yml

```diff
@@ -384,8 +384,9 @@ jobs:
           - config: default
             suite: suite-tpcds
           # this suite is not meant to be run with different configs
-          - config: default
-            suite: suite-oauth2
+          # TODO Re-enable oauth2 product tests - https://github.com/trinodb/trino/issues/8719
```

Comment:
Instead of removing here - can we disable it `SuiteOAuth2` - as an excluded group for tests. CI/CD pipeline would have less history.

Reply:
Then:
 -  it would be a bit more challenging to fix the product tests as they won't hit the issue by default
 - it would make a false impression that ouath2 tests are fixed as they have a green build.

Reply:
Running SuiteOAuth2 without any tests would feel wasteful to me.

Reply:
> CI/CD pipeline would have less history.

What is the problem with history?